libs/wire-api/src/Wire/API/Routes/Version/Wai.hs

```diff
@@ -86,5 +86,5 @@ removeVersionHeader req =
 addVersionHeader :: Version -> Request -> Request
 addVersionHeader v req =
   req
-    { requestHeaders = (versionHeader, toByteString' (fromEnum v)) : requestHeaders req
+    { requestHeaders = (versionHeader, toByteString' (versionInt v :: Int)) : requestHeaders req
```

Comment:
Nice! This crossed my mind the other day. Once we start dropping old versions, we won't be able to rely on the `Enum` instance any more.